plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryPlugin.kt

```diff
@@ -47,12 +55,24 @@ class SentryPlugin : Plugin<Project> {
                 extraProperties.get(SENTRY_PROJECT_PARAMETER).toString()
             }.getOrNull()
 
+            // TODO: this should depend on ignoredVariants/ignoredFlavours/ignoredBuildTypes
+            androidComponentsExtension.onVariants { variant ->
```

Comment:
Could you move it inside the `androidExtension.applicationVariants.matching` block? Or you need to specifically the `.onVariants`?